[hack] add MachineSet creation/deletion script (#121)
openshift/windows-machine-config-operator: README.md

Example MachineSet for other cloud providers:
- [AWS](docs/machineset-aws.md).

> Alternatively, you can use `hack/machineset.sh` script to generate MachineSets for AWS and Azure platform.

Reviewer: nit
```suggestion
Alternatively, the `hack/machineset.sh` script can be used to generate MachineSets for AWS and Azure platforms.
```

The hack script will generate a `MachineSet.yaml` file which can be edited before using or can be used as it is.
The script takes optional arguments like `apply` and `delete` to directly create/delete MachineSet on the cluster without 

> generating a `yaml` file.

Author: if the optional argument is provided, it'll directly interact with cluster and will not create a `yaml` file... 
if the optional argument is not provided, it'll create a `MachineSet.yaml` which the user can edit before applying.. 
this is considering that not everyone will be editing the `MachineSet` before applying(because they might just be testing it with one VM and can easily scale up machineset from console or terminal) and providing argument will be faster and wont require cleaning up created files afterwords...
